ansible/files/configuration/configuration.json

```diff
@@ -20,9 +20,11 @@
     "Properties": {
       "spark.local.dir": "/mnt,/mnt1",
       "spark.driver.maxResultSize": "4g",
+      "spark.driver.memory": "4g",
```

Comment:
That value is not dynamically adjusted and defaults to 1 GB regardless of instance type.